features/steps/cleaner_storage.py

```diff
@@ -320,9 +320,9 @@ def delete_all_tables(context):
             raise e
 
 
-@when(u"I insert following records into database")
+@when(u"I insert following records into REPORT table")
```

Comment:
Just FYI and in case you see it worth it, you can use `@when(u"I insert following records into {TABLE} table")` and then 
```python
insertStatement = ("INSERT INTO {}.......;".format(table))
```

Reply:
then you'll have to construct the rows and values somehow